core/drlibc/drlibc.h

```diff
@@ -72,7 +72,11 @@ dynamorio_mach_syscall(uint sysnum, uint num_args, ...);
 #    else
 ptr_int_t
 dynamorio_syscall(uint sysnum, uint num_args, ...);
-/* N.B. func must not return. */
+/* Wrapper for clone().
+ * N.B. func must not return.
+ * On x86 (32 and 64-bit) this supports passing NULL for newsp which is just
+ * shorthand for the child using the same value as the parent.
```

Comment:
Please add TODO or XXX #6514 comments to the other dynamorio_clone routines on other arches.

Reply:
Done